core-tests/shared/src/test/scala-3/zio/prelude/DerivationSpec.scala

```diff
@@ -0,0 +1,31 @@
+package zio.prelude
+
+import zio._
+import zio.test._
+
+object DerivationSpec extends ZIOSpecDefault {
+
+  final case class Person(name: String, age: Int)
+
+  sealed trait Color
```

Comment:
How come you don't need the `derives Equal` clauses? I though that you need to use them :thinking: 

https://docs.scala-lang.org/scala3/reference/contextual/derivation.html

Reply:
Apparently not.